src/routes/JobBoard/JobBoard.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react'
+import JobCard from '@/components/JobCard/JobCard'
+import { Container } from '@mui/material'
+import { jobData } from '@/utils/jobData'
+
+const JobBoard:FC = () => {
+    return (
+        <Container style={{ padding: 25 }}>
+            {jobData.map((job, index) => (
+                <JobCard
+                    key={index}
+                    title={job.title}
```

Comment:
Rather than unpacking the entire job and passing it into various parameters, let's create a `JobListing` type and pass the object in. Please define the type under the `types` folder.